packages/bus-rdf-join/lib/ActorRdfJoin.ts

```diff
@@ -544,11 +556,6 @@ export interface IActorRdfJoinInternalOptions {
   requiresVariableOverlap?: boolean;
 }
 
-/**
- * Represents a logical join type.
- */
-export type LogicalJoinType = 'inner' | 'optional' | 'minus';
```

Comment:
Be careful: this will be a breaking change.

Reply:
Right, I added the following line: `// TODO remove this in next major version
export { LogicalJoinType } from '@comunica/types';`.
Is that ok? I figured moving it would be better as it it also used by the `iterator-transform` actors.

Reply:
Yep, something like that is ok!
I don't see it in the diff yet though, so I assume it still needs to be pushed?

Reply:
I added it at line 593, is that ok?

Reply:
It is [here](https://github.com/comunica/comunica/blob/332ad8b1ce2f9ccf5f9ff1fb24815b88b4b1356c/packages/bus-rdf-join/lib/ActorRdfJoin.ts#L593), not sure how to post the permalink to the diff, but it does show up for me.

Reply:
Looks perfect!